angular-hometask/src/app/core/breadcrumbs/breadcrumbs.component.html

```diff
@@ -1 +1,6 @@
-<p>Courses</p>
+<ul class="breadcrumb-container">
+    <li class="breadcrumb-list-item"><a href="/" class="breadcrumb">Courses</a></li>
```

Comment:
Maybe in scope of this courses it's good, but it not good solution to leave mock title

Reply:
Not sure what you mean. Guess I have to change the whole logics of breadcrumbs to fix it.